server/model/secret.go

```diff
@@ -105,11 +104,11 @@ func (s Secret) IsRepository() bool {
 
 // Match returns true if an image and event match the restricted list.
 func (s *Secret) Match(event WebhookEvent) bool {
-	if len(s.Events) == 0 {
-		return true
+	if event == EventPullClosed {
+		event = EventPull
 	}
-	for _, pattern := range s.Events {
-		if match, _ := filepath.Match(string(pattern), string(event)); match {
```

Comment:
why was there a filepath match in the first place?!?

Reply:
never mind it's just a droneCI legacy nonesence

https://github.com/woodpecker-ci/woodpecker/blame/f1e2f3dcad3c5eff9255f58cf8eb0430cd80880f/server/model/secret.go#L70-L73